src/main/java/data/TaskManager.java

```diff
@@ -437,8 +507,8 @@ private void handleMarkingOfTask(int taskIndex, LocalDate date) {
      * @throws TaskManagerException If there is an error in managing tasks.
      * @throws DateTimeParseException If there is an error parsing the date.
      */
-    public void priorityManager(WeekView weekView, MonthView monthView, boolean inMonthView, String day, 
-            int taskIndex, String priorityLevelString) 
+    public void priorityManager(WeekView weekView, MonthView monthView, boolean inMonthView, String day,
+                                int taskIndex, String priorityLevelString)
```

Comment:
same as above

Reply:
Consider changing the spacing to what Clarence has highlighted it to be